doc/contributing/making_changes_to_ruby.md

```diff
@@ -2,12 +2,16 @@
 
 Most Ruby documentation lives in the source files and is written in [rdoc format](https://ruby.github.io/rdoc/RDoc/Markup.html).
 
-To generate the output of documentation changes in HTML in the `rdoc/` directory, run
+Some guidelines live under the `doc` folder and can be written in either `rdoc` or `markdown` format, determined by the file extension.
+
+To generate the output of documentation changes in HTML in the `{build folder}/.ext/html` directory, run
```

Comment:
I feel that we should mention it's under the build folder (based on previous steps, it should be `ruby/build`). Otherwise it's confusing to beginners like me.